rest/usage_reporting/report_usage.py

```diff
@@ -15,6 +15,22 @@ def __init__(self):
         self.auth_client_secret = os.environ.get("USAGE_REPORTING_AUTH_CLIENT_SECRET")
         self.base_url = os.environ.get("USAGE_REPORTING_BASE_URL")
 
+        if self.auth_url is None:
```

Comment:
This should be a part of https://git.sinergise.com/team-6/openeo-platform/-/issues/251

Reply:
it was done in a separate MR